testutil/random.go

```diff
@@ -656,27 +657,27 @@ func RandomCheckpoint() *eth2p0.Checkpoint {
 }
 
 func RandomEpoch() eth2p0.Epoch {
-	return eth2p0.Epoch(rand.Uint64())
+	return eth2p0.Epoch(rand.Int63n(int64(math.Pow(2, 53))))
 }
 
 func RandomSlot() eth2p0.Slot {
-	return eth2p0.Slot(rand.Uint64())
+	return eth2p0.Slot(rand.Int63n(int64(math.Pow(2, 53))))
```

Comment:
Note: Had to use `2^53` as boundary for these random values to resolve errors from lodestar.
![image](https://user-images.githubusercontent.com/37813203/234087697-db95c4d5-0d54-406a-95ea-28ded72d0343.png)